pkg/operator/ceph/object/rgw.go

```diff
@@ -281,7 +281,16 @@ func (c *clusterConfig) deleteStore() error {
 
 		// Delete the realm and pools
 		objContext := NewContext(c.context, c.store.Name, c.store.Namespace)
-		err := deleteRealmAndPools(objContext, c.store.Spec)
+		realmName, zoneGroupName, zoneName, err := GetMultisiteForObjectStore(c.context, c.store)
```

Comment:
@leseb @travisn `GetMultisiteForObjectStore` is what I'm doing to get the realm, zg, zone to put into the objectContext. What do you think?

Reply:
Instead of this:
```
objContext := NewContext(c.context, c.store.Name, c.store.Namespace)
realmName, zoneGroupName, zoneName, err := GetMultisiteForObjectStore(c.context, c.store)
```
and then assigning the fields to the context, how about this?
```
objContext, err := NewMultisiteContext(c.context, c.store)
```